gomock/matchers.go

```diff
@@ -382,6 +410,22 @@ func Not(x any) Matcher {
 	return notMatcher{Eq(x)}
 }
 
+// Regex checks whether parameter matches the associated regex.
+//
+// Example usage:
+//
+// Regex("[0-9]{2}:[0-9]{2}").Matches("23:02") // returns true
+// Regex("[0-9]{2}:[0-9]{2}").Matches([]byte{50,51,58,48,50}) // returns true
```

Comment:
```suggestion
// Regex("[0-9]{2}:[0-9]{2}").Matches([]byte{'2', '3', ':', '0', '2'}) // returns true
```
Rationale:
1. Character literals are easier to understand than ASCII character codes in decimal digits.
2. If this were real code, Go would insert spaces after commas.